application/client/src/app/ui/views/toolbar/module.ts

```diff
@@ -2,10 +2,11 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { SearchModule } from './search/module';
 import { HistoryModule } from './history/module';
+import { ChartModule } from './chart/module';
```

Comment:
as soon as Chart component is `@Initial` (should be available for run-time injecting) it should be added also into `bootstrap` section of module
```
@NgModule({
    declarations: [...],
    imports: [...],
    ....
    bootstrap: [ChartModule],
})
```
and in `ChartModule` chart-component also should be included into `bootstrap`.

It will make Angular preload all related to chart stuff on booting stage